src/main/java/com/vaadin/demo/component/basiclayouts/BasicLayoutsHorizontalLayoutWrapping.java

```diff
@@ -0,0 +1,49 @@
+package com.vaadin.demo.component.basiclayouts;
+
+import com.vaadin.demo.DemoExporter; // hidden-source-line
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.orderedlayout.FlexComponent;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.router.Route;
+
+@Route("basic-layouts/horizontal-layout-wrapping")
+public class BasicLayoutsHorizontalLayoutWrapping extends Div {
+
+    public BasicLayoutsHorizontalLayoutWrapping() {
+        HorizontalLayout layoutNoWrap = new HorizontalLayout();
+        layoutNoWrap.setPadding(true);
+        layoutNoWrap.setSpacing(true);
+        layoutNoWrap.setMargin(true);
+        layoutNoWrap.setAlignItems(FlexComponent.Alignment.STRETCH);
+        layoutNoWrap.setWidth("350px");
+        layoutNoWrap.add(new Button("Button 1"));
+        layoutNoWrap.add(new Button("Button 2"));
+        layoutNoWrap.add(new Button("Button 3"));
+        layoutNoWrap.add(new Button("Button 4"));
+        layoutNoWrap.add(new Button("Button 5"));
+        this.add(layoutNoWrap);
+
+        // tag::snippet[]
+        HorizontalLayout layout = new HorizontalLayout();
```

Comment:
```suggestion
        HorizontalLayout layoutWithWrap = new HorizontalLayout();
```
Nit: Would be a bit more consistent and provides some context which of the two layouts it represents.

Reply:
Done for both Java examples.